inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -125,6 +125,10 @@ static const char progress_message[] =
 // @brief message for performance counters option
 static const char pc_message[] = "Optional. Report performance counters.";
 
+// @brief message for switching memory allocation type option
+static const char use_device_mem_message[] =
+    "Optional. Switch beetwen host and device memory allocation for input and output buffers.";
```

Comment:
can we use?
```c++
#if defined OPENCL_FOUND || XXX
#define HAVE_DEVICE_MEM
#endif

#ifdef HAVE_DEVICE_MEM
// @brief message for switching memory allocation type option
static const char use_device_mem_message[] =
    "Optional. Switch beetwen host and device memory allocation for input and output buffers.";
#endif
```

Reply:
I think we can
@vladimir-paramuzov what do you think?

Reply:
The proposal looks good to me

Reply:
BTW, may be we can use more specific `HAVE_GPU_DEVICE_MEM` instead of not-concrete `OpenCL_FOUND`?

Reply:
typo: `beetwen `-> `between`

Reply:
@ilya-lavrenov, thanks, I replaced `OpenCL_FOUND` -> `HAVE_GPU_DEVICE_MEM_SUPPORT`